.travis.yml

```diff
@@ -3,7 +3,6 @@ sudo: false
 python:
   - "2.6"
   - "2.7"
-  - "3.2"
```

Comment:
There is a bug in 3.2's standard library that will cause failure. See [https://github.com/sigmavirus24/requests-toolbelt/issues/88](https://github.com/sigmavirus24/requests-toolbelt/issues/88).